app/src/main/java/com/example/stretchy/features/datatransport/Export.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.stretchy.features.datatransport
+
+import com.example.stretchy.repository.Repository
+import com.google.gson.GsonBuilder
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import java.io.File
+
+class Export(val repository: Repository) : ExportSavedData {
```

Comment:
Interfaces `DataExporter`, `DataImporter`, add `Impl` to name after to implementations